Sources/Swift/Integrations/SessionReplay/SentryBreadcrumbReplayConverter.swift

```diff
@@ -0,0 +1,83 @@
+@_implementationOnly import _SentryPrivate
+import Foundation
+
+@objcMembers
+class SentryBreadcrumbReplayConverter: NSObject {
+    
+    private let supportedNetworkData = Set<String>([
+        "status_code",
+        "method",
+        "response_content_length",
+        "request_content_length",
+        "http.query",
+        "http.fragment"]
+    )
+    
+    func replayBreadcrumbs(from breadcrumbs: [Breadcrumb]) -> [SentryRRWebEvent] {
+        breadcrumbs.compactMap { replayBreadcrumb(from: $0) }
+    }
+    
+    //Convert breadcrumb information into something
+    //replay front understands
+    private func replayBreadcrumb(from breadcrumb: Breadcrumb) -> SentryRRWebEvent? {
+        guard let timestamp = breadcrumb.timestamp else { return nil }
+        if breadcrumb.category == "http" {
+            return networkSpan(breadcrumb)
+        } else if breadcrumb.type == "navigation" {
+            return navigationBreadcrumb(breadcrumb)
+        } else if breadcrumb.category == "touch" {
+            return SentryRRWebBreadcrumbEvent(timestamp: timestamp, category: "ui.tap", message: breadcrumb.message)
+        } else if breadcrumb.type == "connectivity" && breadcrumb.category == "device.connectivity" {
+            guard let networkType = breadcrumb.data?["connectivity"] as? String, !networkType.isEmpty  else { return nil }
```

Comment:
`m`: Why do we drop crumbs for which the networkType is empty?

Reply:
Session replay UI has a specific implementation, any information that is not known for session replay will not be displayed.

Reply:
Again, please add this info as a comment to the code, as it's not obvious.